### ISSUE TITLE
application: consistent behavior when interacting with `select` components

### ISSUE THREAD

**Issue by vince-fugnitto** [microsoft/vscode#152465]
**Details**

Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.68.1
- OS Version: Ubuntu 18.04.6 LTS

**Steps to Reproduce:**

1. start the application
2. open the **debug** view and interact with the debug launch configuration select component - notice that the cursor is a pointer
3. open the **preferences** view and interact with preferences that provide enum values (select components) - notice that the **cursor** is not a pointer

**Description**

In order to improve usability and consistency across the product, the **select** components should behave similarly and display a `cursor: pointer` when interacting with the select components. The individual options provided by the component already display the proper pointer, but not when first interacting with the select.

**Additional Information**

_debug view_: (cursor is a pointer when first interacting with the select)

https://user-images.githubusercontent.com/40359487/174296343-6c65de92-aeec-40d1-be33-f8a3d6f4077e.mp4

_preferences view_: (cursor is not a pointer when first interacting with the select)

https://user-images.githubusercontent.com/40359487/174296387-7b48f921-3e57-474d-b8b7-c666d5341efc.mp4
